databases/db.js

```diff
@@ -64,6 +67,17 @@ Sticker.belongsTo(team,{
   }
 });
 
+//Relaciones entre usuarios y eventos para formar un fantasy
+User.belongsToMany(Event,{
+  through: money
+});
+Event.belongsToMany(User,{
+  through: money
+});
+User.hasMany(money);
+money.belongsTo(User);
+Event.hasOne(money);
```

Comment:
Entiendo que los eventos tienen distintos MONEY así mismo como los jugadores por que es una relacion Many to Many no?

```suggestion
Event.hasMany(money);
```